transformers4rec/torch/features/embedding.py

```diff
@@ -159,8 +159,9 @@ def from_schema(  # type: ignore
         if tags:
             schema = schema.select_by_tag(tags)
 
-        if not item_id and schema.select_by_tag(["item_id"]).column_names:
-            item_id = schema.select_by_tag(["item_id"]).column_names[0]
+        _item_id = schema.select_by_tag(Tags.ITEM_ID)
+        if not item_id and len(_item_id) > 0:
```

Comment:
if for some reason the select_by_tag matches more than one column, maybe would like to raise an exception?